Jetcaster/app/src/main/java/com/example/jetcaster/ui/home/Home.kt

```diff
@@ -179,38 +178,38 @@ fun HomeContent(
                 dominantColorState.reset()
             }
 
-            Stack(Modifier.fillMaxWidth()) {
-                Box(
-                    Modifier.matchParentSize()
-                        .verticalGradientScrim(
-                            color = MaterialTheme.colors.primary.copy(alpha = 0.38f),
-                            startYPercentage = 1f,
-                            endYPercentage = 0f
-                        )
+            Column(
+                modifier = Modifier.fillMaxWidth()
+                    .verticalGradientScrim(
+                        color = MaterialTheme.colors.primary.copy(alpha = 0.38f),
+                        startYPercentage = 1f,
+                        endYPercentage = 0f
+                    )
+            ) {
+                val appBarColor = MaterialTheme.colors.surface.copy(alpha = 0.87f)
+
+                // Draw a scrim over the status bar which matches the app bar
+                Spacer(Modifier.background(appBarColor).fillMaxWidth().statusBarsHeight())
```

Comment:
This makes me want a `.statusBarSize` modifier.

Reply:
Yeahhhh, good shout.